src/dolphin/manager.ts

```diff
@@ -79,17 +79,22 @@ export class DolphinManager {
   }
 
   public async launchNetplayDolphin() {
-    if (this.netplayDolphinInstance) {
-      throw new Error("Netplay dolphin is already open!");
-    }
-
     await this._updateDolphinSettings(DolphinLaunchType.NETPLAY);
 
     const netplayInstallation = this.getInstallation(DolphinLaunchType.NETPLAY);
     const dolphinPath = await netplayInstallation.findDolphinExecutable();
     log.info(`Launching dolphin at path: ${dolphinPath}`);
     const launchMeleeOnPlay = this.settingsManager.get().settings.launchMeleeOnPlay;
-    const meleeIsoPath = launchMeleeOnPlay ? await this._getIsoPath() : undefined;
+    let meleeIsoPath = launchMeleeOnPlay ? await this._getIsoPath() : undefined;
+
+    if (this.netplayDolphinInstance) {
+      if (!launchMeleeOnPlay) {
+        this.netplayDolphinInstance.kill();
```

Comment:
```suggestion
        this.netplayDolphinInstance.kill();
```
This sounds good in theory, but I worry that people could kill their dolphin process A) unintentionally (e.g. they're already in a game or in a training mode or something and we kill it), or B) intentionally by skirting around the game reporting logic in Dolphin (this is already an existing issue).